package.json

```diff
@@ -33,6 +34,7 @@
   },
   "homepage": "https://github.com/digicatapult/dscp-matchmaker-api#readme",
   "devDependencies": {
+    "@digicatapult/dscp-process-management": "^1.6.10",
```

Comment:
Again, we don't need this as an installed dependency.  `npx's` default behaviour is to run a command if its installed in the local package directory and then to download and run it if it doesn't exist.

Reply:
Or is it required for tests?

Reply:
I'll remove